book/preliminary/index.md

```diff
@@ -5,7 +5,7 @@ We welcome all types of learners to our event, from professional software develo
 * navigate a [Jupyter Notebook](https://jupyter.org/) environment
 * conduct file management, text editing and other basic tasks from a command line interface
 * add and commit changes in Git, and push and pull content from GitHub
-* create simple scientific workflows in Python 
+* create simple scientific workflows in Python
```

Comment:
Style question - Should we capitalize the beginning of each bullet point?
For example:
```
* Create simple ...
```

Reply:
Probably... Although not sure how to enforce it other than the manual review